app/src/main/java/io/appium/settings/Settings.java

```diff
@@ -42,6 +44,14 @@ public void onCreate(Bundle savedInstanceState) {
         registerDataConnectionSettingReceiver();
         registerLocationInfoReceiver();
 
+        // https://developer.android.com/about/versions/oreo/background-location-limits
+        if (Build.VERSION.SDK_INT >= 26) {
+            Log.d(TAG, "Initializing the foreground service");
+            Intent intent = new Intent(Settings.this, ForegroundService.class);
+            intent.setAction(ForegroundService.ACTION_START);
+            startService(intent);
```

Comment:
Should we call `startForegroundService` instead of `startSercive`?

Reply:
^ I read https://github.com/appium/io.appium.settings/pull/29/files#diff-8f5c875aba6e2baaddf12cd507c82081R86 and the documentation in the `service` class. I got it.